<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Voucher</class>
 <widget class="QDialog" name="Voucher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>استبدال النقاط بقسيمه شراء</string>
  </property>
  <widget class="QLineEdit" name="Qline_cust">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
                                        background-color: white;
                                        border-style: outset;
                                        border-width: 2px;
                                        border-radius: 10px;
                                        border-color: blue;
                                        min-width: 6em;
                                        }
                                        QLineEdit:hover {
                                        background: qradialgradient(
                                        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
                                        );
                                        }
                                       QLineEdit:pressed {
                                        border-style: inset;
                                        background: qradialgradient(
                                        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
                                        );
                                        }
                                    </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>رقم العميل</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>50</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>اسم  العميل</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Qline_name">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
                                        background-color: white;
                                        border-style: outset;
                                        border-width: 2px;
                                        border-radius: 10px;
                                        border-color: blue;
                                        min-width: 6em;
                                        }
                                        QLineEdit:hover {
                                        background: qradialgradient(
                                        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
                                        );
                                        }
                                       QLineEdit:pressed {
                                        border-style: inset;
                                        background: qradialgradient(
                                        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
                                        );
                                        }
                                    </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Qline_points">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>82</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
                                        background-color: white;
                                        border-style: outset;
                                        border-width: 2px;
                                        border-radius: 10px;
                                        border-color: blue;
                                        min-width: 6em;
                                        }
                                        QLineEdit:hover {
                                        background: qradialgradient(
                                        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
                                        );
                                        }
                                       QLineEdit:pressed {
                                        border-style: inset;
                                        background: qradialgradient(
                                        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
                                        );
                                        }
                                    </string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>90</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>نقاط  العميل</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Qline_replace">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>259</x>
     <y>130</y>
     <width>82</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
                                        background-color: white;
                                        border-style: outset;
                                        border-width: 2px;
                                        border-radius: 10px;
                                        border-color: blue;
                                        min-width: 6em;
                                        }
                                        QLineEdit:hover {
                                        background: qradialgradient(
                                        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
                                        );
                                        }
                                       QLineEdit:pressed {
                                        border-style: inset;
                                        background: qradialgradient(
                                        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
                                        );
                                        }
                                    </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>130</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>عدد النقاط المستبدله</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Qline_remainder">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>82</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
                                        background-color: white;
                                        border-style: outset;
                                        border-width: 2px;
                                        border-radius: 10px;
                                        border-color: blue;
                                        min-width: 6em;
                                        }
                                        QLineEdit:hover {
                                        background: qradialgradient(
                                        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
                                        );
                                        }
                                       QLineEdit:pressed {
                                        border-style: inset;
                                        background: qradialgradient(
                                        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
                                        );
                                        }
                                    </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>170</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>النقاط  المتبقيه</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>210</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>القيمه المستحقه</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Qline_point_value">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>210</y>
     <width>82</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
                                        background-color: white;
                                        border-style: outset;
                                        border-width: 2px;
                                        border-radius: 10px;
                                        border-color: blue;
                                        min-width: 6em;
                                        }
                                        QLineEdit:hover {
                                        background: qradialgradient(
                                        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
                                        );
                                        }
                                       QLineEdit:pressed {
                                        border-style: inset;
                                        background: qradialgradient(
                                        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
                                        );
                                        }
                                    </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Qbtn_search">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
                                        background-color: white;
                                        border-style: outset;
                                        border-width: 2px;
                                        border-radius: 10px;
                                        border-color: blue;
                                        min-width:6em;
                                       }
                                        QPushButton:hover {
                                        background: qradialgradient(
                                        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
                                        );
                                        }
                                        QPushButton:pressed {
                                        border-style: inset;
                                        background: qradialgradient(
                                        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
                                        );
                                        }
                                    
                                    </string>
   </property>
   <property name="text">
    <string>بحث</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Qbtn_replace">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
                                        background-color: white;
                                        border-style: outset;
                                        border-width: 2px;
                                        border-radius: 10px;
                                        border-color: blue;
                                        min-width:6em;
                                       }
                                        QPushButton:hover {
                                        background: qradialgradient(
                                        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
                                        );
                                        }
                                        QPushButton:pressed {
                                        border-style: inset;
                                        background: qradialgradient(
                                        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
                                        );
                                        }
                                    
                                    </string>
   </property>
   <property name="text">
    <string>إستبدال</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Qbtn_print">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
                                        background-color: white;
                                        border-style: outset;
                                        border-width: 2px;
                                        border-radius: 10px;
                                        border-color: blue;
                                        min-width:6em;
                                       }
                                        QPushButton:hover {
                                        background: qradialgradient(
                                        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
                                        );
                                        }
                                        QPushButton:pressed {
                                        border-style: inset;
                                        background: qradialgradient(
                                        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
                                        );
                                        }
                                    
                                    </string>
   </property>
   <property name="text">
    <string>طباعه</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
